libmediation/types/types.h

```diff
@@ -10,6 +11,8 @@
 #endif
 char* strnstr(const char* s1, const char* s2, size_t len);
 
+std::regex invalidChars();
```

Comment:
There's no need to expose this function : it's entirely internal to the implementation of `types.cpp`.